deepdoc/parser/markdown_parser.py

```
@@ -157,9 +159,13 @@ def __init__(self, markdown_content):
         self.lines = markdown_content.split("\n")
 
     def get_delimiters(self, delimiters):
-        toks = re.findall(r"`([^`]+)`", delimiters)
-        toks = sorted(set(toks), key=lambda x: -len(x))
-        return "|".join(re.escape(t) for t in toks if t)
+        # Delegate to the canonical parser (#17383). The previous
+        # implementation matched only backtick-wrapped tokens and dropped
+        # bare characters, which silently made the shipped default
+        # (``\n!?;。；！？``) a no-op for the markdown path. The helper
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)